tests/Fixer/Alias/RandomApiMigrationFixerTest.php

```diff
@@ -221,4 +221,14 @@ public function provideFix73Cases()
             ],
         ];
     }
+
+    public function testFixToRandomInt()
```

Comment:
we already have tests for `random_int`:
- without params: https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/2.19/tests/Fixer/Alias/RandomApiMigrationFixerTest.php#L96-L100
- with params: https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/2.19/tests/Fixer/Alias/RandomApiMigrationFixerTest.php#L163-L167

Reply:
@SpacePossum can you take a look pls?